experiments/mcp-eval/tasks/storagebucket-1-ubla/setup.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+REPO_ROOT="$(git rev-parse --show-toplevel)"
```

Comment:
In future I suspect you can have a default cleanup/setup.  In kubectl-ai I think we were working towards having a "scope: namespace" or "scope: cluster", where we would create a namespace or cluster and clean it up by default.  It feels like here you have "scope: directory"